.ci/pytorch/common_utils.sh

```diff
@@ -196,6 +196,7 @@ function checkout_install_torchdeploy() {
   pushd multipy
   git checkout "${commit}"
   python multipy/runtime/example/generate_examples.py
+  sudo apt-get update && sudo apt-get install -y liblzma-dev
```

Comment:
Why?

Reply:
I suspect we might need it for serialization or something, and previously it was installed as transitive dependency of  libopencv-dev. Please note that this is part of the `multipy` tests, which I didn't know we are still running. Perhaps time to kill those as well.

Reply:
Let's revert some changes and try again.